tests/test_wave_01.py

```diff
@@ -1,7 +1,6 @@
 # The following line imports the Vendor class from the module vendor inside the swap_meet package.
 from swap_meet.vendor import Vendor
 
```

Comment:
Hm... did you mean to commit this file? Remember to use `git status` to check which files are staged/not staged before using `git add .` to add changes! There are times where we don't want to add all changes on files in a project.